network/network_linux.go

```diff
@@ -243,14 +243,24 @@ func isGreaterOrEqaulUbuntuVersion(versionToMatch int) bool {
 	return false
 }
 
-func readDnsInfo(ifName string) (DNSInfo, error) {
-	var dnsInfo DNSInfo
+func readDNSInfo(ifName string) (DNSInfo, error) {
+	var (
+		dnsInfo DNSInfo
+		out     string
+		err     error
+	)
 
 	p := platform.NewExecClient()
-	cmd := fmt.Sprintf("systemd-resolve --status %s", ifName)
-	out, err := p.ExecuteCommand(cmd)
+	resCtlCmd := fmt.Sprintf("resolvectl status %s", ifName)
+	sysResCtlCmd := fmt.Sprintf("systemd-resolve --status %s", ifName)
+
+	out, err = p.ExecuteCommand(resCtlCmd)
 	if err != nil {
-		return dnsInfo, err
+		log.Printf("[net] trying command %s", sysResCtlCmd)
```

Comment:
no need for this log.. `ExecuteCommand` function anyway logs it.. instead can we log error returned by resovleCtlcmd?